kedro/framework/context/context.py

```diff
@@ -166,13 +166,13 @@ class KedroContext:
 
     _package_name: str
     project_path: Path = field(converter=_expand_full_path)
-    config_loader: ConfigLoader
+    config_loader: OmegaConfigLoader
```

Comment:
So the interface is the OmegaConfigLoader and not the AbstractConfigLoader?

Reply:
RTD was causing issues with AbstractConfigLoader but i'm trying to update it

Reply:
I believe this should me the abstract class - some user point this out a while ago but I can't find the issue.